src/main/flight/mixer.h

```diff
@@ -49,32 +49,33 @@
 // Note: this is called MultiType/MULTITYPE_* in baseflight.
 typedef enum mixerMode
 {
-    MIXER_TRI = 1,
-    MIXER_QUADP = 2,
-    MIXER_QUADX = 3,
-    MIXER_BICOPTER = 4,
-    MIXER_GIMBAL = 5,
-    MIXER_Y6 = 6,
-    MIXER_HEX6 = 7,
-    MIXER_FLYING_WING = 8,
-    MIXER_Y4 = 9,
-    MIXER_HEX6X = 10,
-    MIXER_OCTOX8 = 11,
-    MIXER_OCTOFLATP = 12,
-    MIXER_OCTOFLATX = 13,
-    MIXER_AIRPLANE = 14,        // airplane / singlecopter / dualcopter (not yet properly supported)
-    MIXER_HELI_120_CCPM = 15,
-    MIXER_HELI_90_DEG = 16,
-    MIXER_VTAIL4 = 17,
-    MIXER_HEX6H = 18,
-    MIXER_PPM_TO_SERVO = 19,    // PPM -> servo relay
-    MIXER_DUALCOPTER = 20,
-    MIXER_SINGLECOPTER = 21,
-    MIXER_ATAIL4 = 22,
-    MIXER_CUSTOM = 23,
-    MIXER_CUSTOM_AIRPLANE = 24,
-    MIXER_CUSTOM_TRI = 25,
-    MIXER_QUADX_1234 = 26
+    MIXER_UNKNOWN,
+    MIXER_TRI,
+    MIXER_QUADP,
+    MIXER_QUADX,
+    MIXER_BICOPTER,
+    MIXER_GIMBAL,
+    MIXER_Y6,
+    MIXER_HEX6,
+    MIXER_FLYING_WING,
+    MIXER_Y4,
+    MIXER_HEX6X,
+    MIXER_OCTOX8,
+    MIXER_OCTOFLATP,
+    MIXER_OCTOFLATX,
+    MIXER_AIRPLANE,         // airplane / singlecopter / dualcopter (not yet properly supported)
+    MIXER_HELI_120_CCPM,
+    MIXER_HELI_90_DEG,
+    MIXER_VTAIL4,
+    MIXER_HEX6H,
+    MIXER_PPM_TO_SERVO,     // PPM -> servo relay
+    MIXER_DUALCOPTER,
+    MIXER_SINGLECOPTER,
+    MIXER_ATAIL4,
+    MIXER_CUSTOM,
+    MIXER_CUSTOM_AIRPLANE,
+    MIXER_CUSTOM_TRI,
+    MIXER_QUADX_1234,
```

Comment:
I definitely think we should keep the number assignments here, since we may wish to delete values in the middle while preserving compatibility